src/resources/SearchInterfaces/SearchInterfaces.model.ts

```diff
@@ -10,6 +10,10 @@ export interface IAccesses {
      * The list of domains that are allowed to access the search interface.
      */
     domains: string[];
+
+    sharingLinkEnabled?: boolean;
```

Comment:
@jeremierobert-coveo and @agagnon2 , what would be the best description for both props?